api-client/src/main/java/com/xing/api/CallSpec.java

```diff
@@ -172,6 +172,13 @@
      */
     CallSpec<RT, ET> formField(String name, List<String> values);
 
+    /**
+     * Overrides the connection's default timeouts for this particular call.
+     *
+     * All timeouts in seconds.
```

Comment:
Hmm, just because the others don't explain doesn't mean you can't start it. 
Also the others are informing about exceptions that might be thrown, so you should do so as well.